src/daemon/ca/certauth.rs

```diff
@@ -1108,33 +1125,48 @@ impl CertAuth {
             match self.find_parent_rc(&parent_handle, &parent_rc_name) {
                 Some(rc) => {
                     // We have a matching RC, make requests (note this may be a no-op).
-                    res.append(&mut self.make_request_events(&mut version, ent, rc, signer.deref())?);
+                    event_details.append(&mut self.make_request_events(ent, rc, signer.deref())?);
                 }
                 None => {
                     // Create a resource class with a pending key
-                    let key_id = signer.create_key()?;
+                    let pending_key = signer.create_key()?;
```

Comment:
Should this be named `_key_id` as `fn create_key()` returns a key identifier and not a key?